rfcs/0024-ckb-system-script-list/0024-ckb-system-script-list.md

```diff
@@ -247,6 +247,76 @@ and the `out_point` of `anyone_can_pay` is
 }
 ```
 
+### cheque
+
+[cheque](https://talk.nervos.org/t/sudt-cheque-deposit-design-and-implementation/5209) ([Source Code](https://github.com/nervosnetwork/ckb-cheque-script/tree/4ca3e62ae39c32cfcc061905515a2856cad03fd8)) allows a sender to temporarily provide cell capacity in asset transfer.
+
+cheque script is for **lock script**:
+
+- Lina
+
+| parameter   | value                                                                |
+| ----------- | -------------------------------------------------------------------- |
+| `code_hash` | `0xe4d4ecc6e5f9a059bf2f7a82cca292083aebc0c421566a52484fe2ec51a9fb0c` |
+| `hash_type` | `type`                                                               |
+| `tx_hash`   | `0x04632cc459459cf5c9d384b43dee3e36f542a464bdd4127be7d6618ac6f8d268` |
```

Comment:
The output cells 0 in this tx has already been spent. Also the cell has no the type script.

https://explorer.nervos.org/transaction/0x04632cc459459cf5c9d384b43dee3e36f542a464bdd4127be7d6618ac6f8d268

Reply:
This output cell is unspent. See the first transaction in https://explorer.nervos.org/address/ckb1qyqlgpvldxaum4w9pqd9cpkmkem6e3w0mqtqqren4v

Reply:
As I can see, the type of acp dep_group cell is also null.

Reply:
Got it. I thought it was the cheque code cell.